src/routes/+layout.ts

```diff
@@ -1 +1,27 @@
 export const ssr = false;
+
+//
+
+import { r, type Locale } from '$lib/i18n';
+import { redirect } from '@sveltejs/kit';
+import { Device } from '@capacitor/device';
+import { getLanguagePreference, setLanguagePreference } from '$lib/preferences/lang';
+
+const setInitialLanguage = async () => {
+	if (!(await getLanguagePreference())) {
+		const lang = await Device.getLanguageTag();
+		await setLanguagePreference(lang.value.split('-')[0]);
+	}
+};
+
+const getLang = async () => {
+	const lang = await getLanguagePreference();
+	if (lang && ['it', 'en'].includes(lang)) return lang as Locale;
```

Comment:
I propose something like this for now:
```
//@ts-ignore
	if (lang && availableLanguageTags.includes(lang)) return lang as Locale;
```

Reply:
Bravissimo @phoebus-84 ! I guess the getLang is in the root `layout.ts`, so it's executed in all pages, correct?

The very point is to have all the languages enumerated in just one place (maybe inlang have also a dynamic call for that) anyhow the thing to imagine is that maybe tomorrow we'll add the taiwanese and just addind the ch-tw and a .json of translations, should be enough to have the language of the app translated. 
This should __obviously__ also reflected in the APP stores (I know there different fields of the google play store and the Apple Store (I don't know if inlang/ionic allows that by default, but we have to dig it).

ANYHOW the source of truth of the languages, should be just in one place!